### ISSUE TITLE
Auto Tag keeps skipping my MP3 files

### ISSUE THREAD

**Issue by AndreaCan** [Marekkon5/onetagger#98]
I use version 1.3.0 on Windows 10. When I use Auto Tag on some MP3 files I always get that my tracks are skipped. This occurs with any platform (Beatport, Discogs, etc):
![immagine](https://user-images.githubusercontent.com/101992616/159179725-c61b7b85-2582-4bf0-8cc2-01ff5bee84d2.png)

These files have been verified with MP3tag and they have the Title and Artist tags perfectly compiled and readable:
![immagine](https://user-images.githubusercontent.com/101992616/159179821-a79c1646-a044-4e99-ba8c-854f4e78e7a0.png)

I also checked the OneTagger LOG and it seems that there is no track metadata (I am attaching the onetagger.log file)

But if the tracks opened with MP3Tag display the Artist and title tags, then why can't One Tagger be able to read them?

Thanks
Andrea

[onetagger.log](https://github.com/Marekkon5/onetagger/files/8312054/onetagger.log)

**Comment by Marekkon5** [comment]
Hello, I don't see anything wrong in the log, maybe 1T can't read the tag. Could you please send on of the songs as example so we can test? Thanks

**Comment by AndreaCan** [comment]
> I don't see anything wrong in the log

In fact, the LOG shows only that OneTagger is unable to read the tags from the files.

> Could you please send one of the songs as example

Ok I send it as an attachment below.
[DJ Kuba & Neitan & Skytech - Dancing (Wh0 Festival Remix).zip](https://github.com/Marekkon5/onetagger/files/8315417/DJ.Kuba.Neitan.Skytech.-.Dancing.Wh0.Festival.Remix.zip)

**Comment by Marekkon5** [comment]
Hello, the track works fine on the development (master) version. There has been some changes since 1.3.0 release, so your issue probably got resolved as well. You can get the test version from the Actions tab on Github. 

Also the track gets no match - that's because on Beatport the name is `Dancing Wh0 Extended Festival Remix` and yours is `Dancing (Wh0 Extended Festival Remix)`. The word `Extended` makes so big difference the accuracy is below the default strictness (0.8). So either change the strictness or rename your song to match it succesfully on Beatport.

If you have any more issues feel free to comment / reopen. Thanks

**Comment by AndreaCan** [comment]
> the track works fine on the development (master) version.

Well then I await the new version.

> You can get the test version from the Actions tab on Github.

I have tried to figure out how to do this, but could not find the installation file. If I have to compile the source for windows I am not equipped to do it.

> Also the track gets no match - that's because on Beatport the name [...]

This wouldn't be a problem, it can happen. The important thing is that the track is not skipped without motivation.

ok thanks, then I'm waiting for you to publish the new version.

**Comment by Marekkon5** [comment]
You don't have to compile it from source. You just have to be logged into Github, then click the Actions tab on top:

![image](https://user-images.githubusercontent.com/15169286/159360551-d58a0aa9-ce1d-4032-9eec-a06d1c2bbcca.png)

Then click on the latest commit:

![image](https://user-images.githubusercontent.com/15169286/159360580-8b5ddf99-0efc-48ad-ad72-d4005cf099a0.png)


There **scroll to bottom** and you can see the `Artifacts` section:

![image](https://user-images.githubusercontent.com/15169286/159360636-56d6cacf-95a9-48e9-b8dd-4cfdc5016385.png)


If you use Windows - click `onetagger-win-setup` and it will download the Windows Setup version.

**Comment by AndreaCan** [comment]
> You don't have to compile it from source.

Thanks to your help I was able to find the file to download. 

I installed the new version (which in any case is always indicated as 1.3.0 in the home of the program) and I analyzed the previous folder: none are skipped and all are "**Matched**" except the one you said, because there is that difference in the title. However now it works fine, as before. Who knows what the old version had that didn't make the program work.

I thank you again for your work you have done with this very useful program.
